lib/api/src/tests/stories.test.js

```diff
@@ -531,49 +532,31 @@ describe('stories API', () => {
       expect(initialStoriesHash['a--1'].args).toEqual({ a: 'b' });
       expect(initialStoriesHash['b--1'].args).toEqual({ x: 'y' });
 
-      const api = new EventEmitter();
-      init({ api });
-
+      init();
```

Comment:
init call has no arguments anymore, fullAPI is provided in module's construction.